toplevel.py

```diff
@@ -14,4 +14,13 @@
 second-level functions.
 The idea is that you would call these either from a GUI
 or from your notebook directory.
+
+Scripts on top level:
+    proc_CPMG.py
+    proc_echo_mw.py
+    proc_Hahn_echoph.py
+    proc_IR.py
+    proc_IR_noecho.py
+    proc_nutation.py
+    proc_square_refl.py
```

Comment:
I would explain that these will be converted into top level functions

Reply:
done